packages/@magic-sdk/react-native-bare/src/index.ts

```diff
@@ -69,3 +69,5 @@ export type Magic<T extends MagicSDKExtensionsOption<any> = MagicSDKExtensionsOp
   SDKBaseReactNative,
   T
 >;
+
+export { useInternetConnection } from './hooks';
```

Comment:
This is close, but not exactly what I believe we were hoping for to resolve this issue. When we spoke, I was under the impression that the idea was that we'd utilize something like this within the SDK and return an error to the developer. From what I can tell [here](https://github.com/magiclabs/magic-js/pull/665/files#diff-c7d7b729e018c26c53125262037c912483ac4ac2f49171243549d87035c8040cR88-R113), the developer would need to add boiler plate around the `<m.relayer />` to detect the internet failure. 

Have you considered potentially adding the check for internet before [this invocation](https://github.com/magiclabs/magic-js/blob/e7ab328946259baa518e43ddf4e010b11df1da08/packages/%40magic-sdk/react-native-expo/src/index.ts#L47-L66)?  Will RN spit out a readable error if a component like `<m.relayer />` `throw`s an error? LMK your thoughts here.